src/test/resources/alma/(DE-605)HT003176544.json

```diff
@@ -94,6 +94,62 @@
     } ]
   },
   "subject" : [ {
+    "type" : [ "Concept" ],
+    "label" : "Berlin",
```

Comment:
as argued in the mother issue and as mentioned in the standup today, don't type it as skos concept. It's lacking at least the most crucial `id`.